exercises/isogram/cases_test.go

```diff
@@ -49,6 +49,11 @@ var testCases = []struct {
 		input:       "thumbscrew-japingly",
 		expected:    true,
 	},
+	{
+		description: "hypothetical word with duplicated character after hyphen",
+		input:       "thumbscrew-jappingly",
+		expected:    false,
+	},
```

Comment:
Looks like an excellent test case to add.  Thanks for the feedback from a student/mentor experience.